### ISSUE TITLE
Add has_spoiler to InlineQueryResult

### ISSUE THREAD

**Issue by Sarisan** [tdlib/telegram-bot-api#558]
If it's possible can we have `has_spoiler` in InlineQueryResult objects like InlineQueryResultGif, InlineQueryResultMpeg4Gif, InlineQueryResultPhoto, InlineQueryResultVideo?

**Comment by levlam** [comment]
Yes, this is possible, but it should be supported first by apps and server-side.
